test/math.jl

```diff
@@ -919,7 +913,7 @@ end
         @test_throws DomainError acosh(T(0.1))
         @test acosh(one(T)) === zero(T)
         @test isnan_type(T, acosh(T(NaN)))
-        for x in (nextfloat(T(1)), pcnfloat(T(2))..., pcnfloat(T(2^28))...)
+        for x in Iterators.flatten(pcnfloat.([T(1) T(2), T(2^28)]))
```

Comment:
Shouldn't it be `Iterators.flatten([nextfloat(T(1)); pcnfloat.([T(2), T(2^28)])`?

Reply:
I seem to have pushed without this fix.